source/blog/2018-06-29-ember-3-2-released.md

```diff
@@ -0,0 +1,295 @@
+[ember-try](https://github.com/ember-cli/ember-try) addon is a great way to
+continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.2
+
+Ember.js 3.2 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is one (1) new feature and three (3) deprecations in this version.
+
+#### New Features (1)
+
+##### Block let template helper (1 of 1)
+
+The new `let` template helper makes it possible to create new bindings in templates. It is like `with` but without the conditional rendering of the block depending on values passed to the block.
+
+Let's say we need to capitalize the first name and last name in our template. We could do something like this:
+
+```handlebars
+Welcome back {{concat (capitalize person.firstName) ' ' (capitalize person.lastName)}}
+
+Account Details:
+First Name: {{capitalize person.firstName}}
+Last Name: {{capitalize person.lastName}}
+```
+
+This could result in an error since we have to keep track of this throughout the template. Thankfully, this is now easier with the `let` helper:
+
+```handlebars
+{{#let (capitalize person.firstName) (capitalize person.lastName)
+  as |firstName lastName|
+}}
+  Welcome back {{concat firstName ' ' lastName}}
+
+  Account Details:
+  First Name: {{firstName}}
+  Last Name: {{lastName}}
+{{/let}}
+```
+
+Now you can use `firstName` and `lastName` inside the `let` block with the comfort of knowing that the logic is in a single place. This is a neat way of introducing bindings in your templates without making them properties on the controller or component.
+
+What is important to know about the `let` helper is that it only works as a block helper. This means that you cannot do like this:
+
+```handlebars
+{{let
+  firstName=(capitalize person.firstName)
+  lastName=(capitalize person.lastName)
+}}
+```
+
+#### Deprecations (3)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+##### Use of Ember.Logger (1 of 3)
```

Comment:
This doesn't seem to be rendering as a header. Is H5 (`#####`) a thing in this markdown processor?

Reply:
it's rendering but it's not styled. I'll do a different PR to the website CSS to make sure we have styling for h5 - thanks for catching that! :+1: